### PR TITLE
Remove extra `get` on secrets

### LINKED ISSUE #1: `get_auth_info` in `chaosreliably.__init__.py` tries to reference a non-existent key
When an experiment with a top level entry of:

```json
"secrets": {
  "reliably": {
    "host": "a-host"
  }
}
```

And action entry of:

```json
"provider": {
    "type": "python",
    "module": "chaosreliably.slo.probes",
    "func": "get_last_N_slos",
    "arguments": {
        "quantity": 1
    },
    "secrets": ["reliably"]
},
```

Is provided, the `get_auth_info` function mistakenly tries to do:

```python
reliably_secrets = secrets.get("reliably", {})
```

This *always* results in `reliably_secrets={}` because `secrets={"host": "a-host"}`.

The block here https://github.com/chaostoolkit-incubator/chaostoolkit-reliably/blob/master/chaosreliably/__init__.py#L81-L83 should instead be:

```python
secrets = secrets or {}
reliably_token = secrets.get("token")
reliably_host = secrets.get("host")
```

### PR DESCRIPTION
**Pull request by ciaransweet**
# What I am changing

TL;DR: This PR fixes an issue where we were selecting `reliably` from secrets when the `reliably` key was no longer available
due to how CTK passes secrets to actions

Closes #1

# How I did it

* `chaosreliably/__init__.py`:
  * Removed `get('reliably')`
* `tests/test_auth.py`:
  * Changed test secrets to correctly represent the form they arrive in
* `CHANGELOG.md`:
  * Fix broken links  
  
# How you can test it

Setup the project for development and run `pytest`

Signed-off-by: Ciaran Evans <ciaran@reliably.com>

### CODE REVIEW

**Review by Lawouach — APPROVED** [review]
LGTM

Usually you also have a changelog entry.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,28 +2,32 @@
 
 ## [Unreleased][]
 
-[Unreleased]: https://github.com/chaostoolkit/chaostoolkit-reliably/compare/0.1.2...HEAD
+[Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-reliably/compare/0.1.2...HEAD
```
Comment (Lawouach) [inline CHANGELOG.md:5]: good catch

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,28 +2,32 @@
...
+
+### Changed
+
+- Fix issue where secrets were not correctly extracted from Experiment config [#1](https://github.com/chaostoolkit-incubator/chaostoolkit-reliably/issues/1)
```
Comment (Lawouach) [inline CHANGELOG.md]: we usually do it this way:

```markdown
- Fix issue where secrets were not correctly extracted from Experiment config [#1][1]

[1]: https://github.com/chaostoolkit-incubator/chaostoolkit-reliably/issues/1)
```

**Review by Lawouach — APPROVED** [review]
LGTM
